algorithms/openie-stanford/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "core_nlp_server_url": "http://nlp-openie-stanford.azurewebsites.net",
+  "core_nlp_server_url": "http://13.79.167.74",
```

Comment:
Should we really have the url here? I think env variables should be there as well.